packages/headless/src/controllers/facets/range-facet/date-facet/date-range.ts

```diff
@@ -73,6 +77,15 @@ export function buildDateRange(config: DateRangeOptions): DateRangeRequest {
 
 function buildDate(rawDate: DateRangeInput, options: DateRangeOptions) {
   const {dateFormat, useLocalTime} = options;
+  if (isRelativeDate(rawDate)) {
+    const localTime = useLocalTime ? {useLocalTime: true} : {};
+    return formatRelativeDate({...localTime, ...rawDate});
+  }
+
+  if (typeof rawDate === 'string' && isRelativeDateFormat(rawDate)) {
```

Comment:
Allows for directly passing the correct string format as a date input, could be useful for SSR...

Reply:
Hmm, why would it be useful for SSR?

Personally, I would prefer to have one way to do things, forcing devs to configure dates using an object. This would provide auto-complete, and error checking at compile-time as opposed to run-time as in the case of a string.

Reply:
Now with this PR, I see the relative date string format as a supported Headless `string` format, you can pass it directly in the facet's currentValues and it should work.

This buildDateRange tool already accepted string/number/Date. I dont' see it as wrong to allow the Headless string format here. If we don't, some might see it as an issue.

For SSR, would we have currentValues with the relative date string format? If thats the case, this method is used to validate manualRanges, if we don't support it, it will cause a bug.